report-hazard/report-hazard.js

```diff
@@ -0,0 +1,96 @@
+const allPages = document.querySelectorAll('div.page');
+allPages[0].style.display = 'block';
+
+function navigateToPage(event) {
```

Comment:
rename `navigateToPage` to `displayCurrentSection`

Reply:
commit eacfaa3f53650861c0c64ecb7218cdfd0a9d9bc3